protobufs/livekit_rtc.proto

```diff
@@ -479,6 +534,9 @@ message RequestResponse {
     QUEUED = 4;
     UNSUPPORTED_TYPE= 5;
     UNCLASSIFIED_ERROR = 6;
+    INVALID_HANDLE = 7;
+    INVALID_NAME = 8;
+    NAME_TAKEN = 9;
```

Comment:
Sorry @ladvoc . Upon further thought, can we just collapse this to one reason `INVALID_PARAM`. Thinking we can keep it generic and provide a more detailed reason in `message` field below.

Reply:
Yeah, that's probably ideal since duplicate handle is an internal error vs. duplicate name is a client error. I will change it.